these-de-medecine-paris-xi.csl

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="fr-FR">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>These de medecine Paris XI</title>
+    <id>http://www.zotero.org/styles/these-de-medecine-paris-xi</id>
+    <link href="http://www.zotero.org/styles/these-de-medecine-paris-xi" rel="self"/>
+    <link href="http://www.zotero.org/styles/medecine-sciences" rel="template"/>
+    <link href="http://www.zotero.org/styles/plos" rel="template"/>
+    <link href="http://www.medecinesciences.org" rel="documentation"/>
```

Comment:
The only documentation I have is a document given by the university. It is in French.
I uploaded it on a personal cloud (http://ovh.to/hCBLzv6) but is there a common repository for this kind of document ? Should I commit it in this repository ?